bindings/python/unicorn/unicorn.py

```diff
@@ -424,6 +424,9 @@ class uc_arm64_cp_reg(ctypes.Structure):
         ("val", ctypes.c_uint64)
     ]
 
+    def __repr__(self):
```

Comment:
If you would like to add `__repr__` for `uc_arm64_cp_reg`, could you do the same for `uc_arm_cp_reg`?